### PR TITLE
docs(integrations): Custom Social Login Key

### PR DESCRIPTION
**Pull request by revant**
documentation about custom social login key

### CODE REVIEW

**Comment by revant** [comment]
https://github.com/frappe/frappe_io/issues/257

**Comment by ddavidebor** [comment]
Waiting on #283 before evaluating merging

File: frappe_io/www/docs/user/en/guides/integration/social_login_key.md [hunk 1]
```diff
@@ -24,7 +24,25 @@ e.g. Social Login Key
...
 - **Client Secret** : `123456`
 - **Enable Social Login** : `Check`
-- **Base URL** : `https://erpnext.org` (required for some providers)
+- **Base URL** : `https://erpnext.org` (required for self hosted providers)
+
+#### Setup Custom Social Logins
+
+There is a standard callback that can be set as redirect url for custom social logins 
+
+redirect url is to be set as `/api/method/frappe.integrations.oauth2_logins.custom/+<doc.name>`
```
Comment (ddavidebor) [inline frappe_io/www/docs/user/en/guides/integration/social_login_key.md:33]: Please improve english and grammar here

**Comment by revant** [comment]
follow up here https://github.com/frappe/frappe_io/pull/165

**Pull request closed without merging**